### DISCUSSION TITLE
Deploying ML models

### DISCUSSION THREAD

**Discussion by Itachiuchiha123** [community/community#128156] [Programming Help] [answered]
### Body

ive recently been doing some courses on machine learning and have developed some simple machine learning models, and i need help deploying these models .

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Uknowme-h** [comment, marked as answer]
since you will already know python , you can use flask and docker to deploy your model  or you can also use streamlit or gradio to deploy your model on [huggingface](https://huggingface.co/) (you can use the api from gradio interface to make your own webapp using javascript or python)

**Reply by ebndev** [reply]
@Uknowme-h thanks for taking the time to answer!

@Itachiuchiha123 best of luck on your ML journey 🌟
